modules/admin_manual/pages/installation/manual_installation/server_prep_ubuntu_20.04.adoc

```diff
@@ -22,6 +22,7 @@ version lower than 7.4, your PHP version is either already EOL or, for PHP 7.3,
 in 2021. Consider upgrading to PHP 7.4. For more information see
 {php_versions_url}[Currently Supported PHP Versions]. This document considers two basic scenarios.
 You have/plan to upgrade from 18.04 LTS to 20.04 LTS or you have a fresh Ubuntu 20.04 LTS installation.
+Php 8.0 is not supported by ownCloud 10.7 server.
```

Comment:
```suggestion

IMPORTANT: PHP 8.0 is not supported by ownCloud 10.7 server.
```

Reply:
I assume we won't support php 8 in the next releases either.

Reply:
We don't support PHP 8 in ownCloud core 10.0 ... 10.7 and won't support it in the upcoming 10.8.

Support might come in the future - that is yet to happen, and when that happens we will need to look ffor these sort of statements and work out what words to rewrite.